packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -76,7 +116,39 @@ const ComboboxNoResults = ({
   );
 };
 
-const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
+function nextToFocus(
+  elements: HTMLElement[],
+  fallback: HTMLElement,
+  focusedIdx: number,
+  direction: number
+): HTMLElement | null {
+  const elems = elements.concat(fallback);
+  if (direction === -1) {
+    return null;
+  }
+
+  const style = getComputedStyle(elems[focusedIdx]);
```

Comment:
Generally speaking, we want to try to avoid calling things that may result in [layout or reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a) unless it's absolutely necessary. It might not be a problem here since it's only being invoked on `keydown` but it's not immediately clear why we need to know anything about layout/positioning.